srcutil/lempar.c

```diff
@@ -22,16 +22,13 @@
 ** The following is the concatenation of all %include directives from the
 ** input grammar file:
 */
-#include <stdio.h>
 /************ Begin %include sections from the grammar ************************/
 %%
 /**************** End of %include directives **********************************/
-/* These constants specify the various numeric values for terminal symbols
-** in a format understandable to "makeheaders".  This section is blank unless
-** "lemon" is run with the "-m" command-line option.
-***************** Begin makeheaders token definitions *************************/
+/* These constants specify the various numeric values for terminal symbols.
```

Comment:
Do you know what is the relation between **srcutil/lempar.c** and **src/query_parser/template.c.tpl**

I see in coord/src/rmr/redise_parser/Makefile
`lemon -s -Ttemplate.c.tpl grammar.y`

And in srcutil/make-parser.mk
`TEMPLATE := $(SRCUTIL)/lempar.c`
and
`$(LEMON) -l -s -T$(TEMPLATE) parser.y`

Reply:
`template.c.tpl` looks like a modified version of `lempar.c` used in `coord/src/rmr/redise_parser` (only referred to inside `coord/src/rmr/redise_parser\Makefile`). This looks like something very dark. Let's look at it carefully.

Reply:
Should this effect this PR? I'm not sure I understand the differences between the files.